infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/definitions/user/NoQuotaAvailable.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.service.smartevents.infra.exceptions.definitions.user;
+
+import javax.ws.rs.core.Response;
+
+public class NoQuotaAvailable extends ExternalUserException {
+
+    private static final long serialVersionUID = 1L;
+
+    public NoQuotaAvailable(String message) {
+        super(message);
+    }
+
+    public NoQuotaAvailable(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    @Override
+    public int getStatusCode() {
+        return Response.Status.NOT_ACCEPTABLE.getStatusCode();
```

Comment:
Is there any specific reason to return HTTP 406?
Based on https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors it looks ike this error code is related to Accept header.

Reply:
fine for me